hudi-client/src/test/java/org/apache/hudi/client/TestHoodieClientBase.java

```diff
@@ -16,8 +16,10 @@
  * limitations under the License.
  */
 
-package org.apache.hudi;
+package org.apache.hudi.client;
 
+import org.apache.hudi.common.HoodieClientTestHarness;
+import org.apache.hudi.WriteStatus;
```

Comment:
ditto, can we place it into e.g. `org.apache.hudi.write.common.WriteStatus`?

Reply:
Actually, top level API classes e.g SparkContext can be placed under the root package right.?  That was my reasoning at-least.. This is the object we return back to the user and made sense to leave at the top level. WDYT

Reply:
Actually, I agree with the context like `SparkContext` can be placed under the top-level package.  However, IMO, the `WriteStatus` is not as important as `SparkContext`. It is not been used anywhere. Hudi support `write` and `read` operation. We should make them a pair relationship.

Reply:
>It is not been used anywhere. 

RDD level APIs have return types like this and projects like uber's marmaray , hudi's deltastramer do use them. Since, you feel strongly, let me move it. :)